org.flatpak.gnomeConnectionManager.yml

```diff
@@ -0,0 +1,59 @@
+app-id: org.flatpak.gnomeConnectionManager
```

Comment:
this app-id is invalid. `org.flatpak` is reserve for flatpak. Use something in relation to the code hosting (github) if the project doesn't have a domain.